bin/tools.py

```diff
@@ -1216,10 +1216,8 @@ def run_parsed_arguments(args):
                         web=True,
                     )
                 # Only build the problem slides if at least one problem has the TeX for it
-                if any(
-                    glob(problem.path / "problem_statement", "problem-slide.*.tex")
-                    for problem in problems
-                ):
+                slideglob = latex.PdfType.PROBLEM_SLIDE.path("*")
+                if any(problem.path.glob(slideglob) for problem in problems):
```

Comment:
Here, the glob is still a `Path` and not converted to `str()`. Does that mean we also don't need it in other places?

Reply:
i think it depends on the glob function we are using...